ksql-engine/src/main/java/io/confluent/ksql/codegen/SqlToJavaVisitor.java

```diff
@@ -125,6 +128,7 @@ private String formatExpression(final Expression expression) {
         final StringLiteral node,
         final Boolean unmangleNames
     ) {
+      functionArguments.addArgumentType(Schema.Type.STRING);
```

Comment:
Don't we need to add a call like this for BOOLEAN?

Reply:
yep - thanks